app/build.gradle

```diff
@@ -37,11 +37,11 @@ repositories {
 
 dependencies {
     compile fileTree(dir: 'libs', include: ['*.jar'])
-    compile 'com.android.support:appcompat-v7:23.1.1'
-    compile 'com.android.support:cardview-v7:23.1.1'
-    compile 'com.android.support:recyclerview-v7:23.1.1'
-    compile 'com.android.support:design:23.1.1'
-    compile 'com.android.support:palette-v7:23.1.1'
+    compile 'com.android.support:appcompat-v7:23.4.0'
+    compile 'com.android.support:cardview-v7:23.4.0'
+    compile 'com.android.support:recyclerview-v7:23.4.0'
+    compile 'com.android.support:design:23.4.0'
+    compile 'com.android.support:palette-v7:23.4.0'
     compile 'com.squareup.picasso:picasso:2.5.2'
     compile 'org.jbundle.util.osgi.wrapped:org.jbundle.util.osgi.wrapped.org.apache.http.client:4.1.2'
     compile('com.github.afollestad.material-dialogs:core:0.8.5.2@aar') {
```

Comment:
At this point, you may want to look at:
-Updating gradle to 2.2.1
-Updating gradle wrapper to 2.14.1 or 3.1
-Updating compileSdkVersion to 24, buildToolsVersion to 24.0.3, and targetSdkVersion to 24
-Updating support libraries to 24.2.1
-Updating com.github.afollestad.material-dialogs:core to 0.9.0.2
-Updating com.github.afollestad:async to 0.2.3
-Updating com.crashlytics.sdk.android:crashlytics to 2.6.5